src/php/Command/ReplCommand.php

```diff
@@ -55,24 +55,31 @@ public function run(): void
     private function loopReadLineAndAnalyze(): void
     {
         while (true) {
-            $this->io->output("\e[?2004h"); // Enable bracketed paste
-            $input = $this->io->readline('>>> ');
-            $this->io->output("\e[?2004l"); // Disable bracketed paste
-            $this->checkInputAndAnalyze($input);
+            try {
+                $this->io->output(self::ENABLE_BRACKETED_PASTE);
+                $input = $this->io->readline(self::PROMPT);
+                $this->io->output(self::DISABLE_BRACKETED_PASTE);
+
+                try {
+                    $this->checkInputAndAnalyze($input);
+                } catch (ExitException $e) {
+                    break;
+                }
+            } catch (Throwable $e) {
+                $this->io->output($e->getTraceAsString() . PHP_EOL);
```

Comment:
Maybe I'm wrong. But `getTraceAsString` only returns the stack trace, but not the message? Both would be good :)

Reply:
This is what you see with the current code. I think the error is there:  `PHP Notice: undefined index ... etc`

<img width="1550" alt="Screenshot 2021-01-22 at 17 26 38" src="https://user-images.githubusercontent.com/5256287/105517166-ffd77880-5cd6-11eb-89b2-64b6e04cb916.png">

Reply:
Ok. Thanks